pyspecProcScripts/simple_functions.py

```diff
@@ -1,23 +1,31 @@
 "First order functions for very simple (a few lines) data manipulation"
 import numpy as np
-def select_pathway(s,pathway,mult_ph_dims=False):
+def select_pathway(*args,**kwargs):
     r"""select a particular CT pathway from the signal `s`
+
+    Arguments are *either* ``pathway`` -- a dict of key/value pairs indicating
+    the pathway **or** the same set of key/value pairs, just passed as a dict.
     
     Parameters
     ==========
+    s: nddata
+        the data whose coherence pathway you would like to select
+
     pathway: dict
         keys are the names of the coherence transfer dimensions (conj. of phase
         cycling dimensions) and values are the pathway you want to select
     """
-    retval = s
-    if mult_ph_dims:
-        for k, v in pathway.items():
-            retval = retval[k,v]
-        return retval
+    if len(args) == 2 and len(kwargs) == 0:
+        s,pathway = args
+    elif len(args) == 1 and len(kwargs) > 0 and len(kwargs)%2 == 0:
+        s = args[0]
+        pathway = kwargs
     else:
-        for k in pathway.items():
-            retval=retval[k]
-        return retval    
+        raise ValueError("your arguments don't make any sense!!")
+    retval = s
+    for k, v in pathway.items():
+        retval = retval[k,v]
+    return retval
```

Comment:
I didn't understand this multi_ph_dims arg, and it's only used in one spot.  I got rid of that and modified to allow the user to pass either a dictionary (as we have been doing), or just key/value kwargs, like ``select_pathway(data,ph1=1,ph2=-2)`` --> if that's not clear, please feel free to mod docstring.

Reply:
this looks fine